modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -391,7 +409,7 @@ function parseUint64Values(
   return values;
 }
 
-function parsePositions(attribute, tile) {
+function parsePositions(attribute: NormalizedAttribute, tile: Tile): Matrix4 {
```

Comment:
Why do you have special naming for attributes that require normalization? FYI `positions` is not normalized attribute